### PR TITLE
Adding CONTRIBUTING.md file

### PR DESCRIPTION
**Pull request by allevin**
Create the CONTRIBUTING.md file with info for forked development and general updates.

This has all the changes in a squashed commit.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+## How to contribute to SST-Macro
+
+#### **Source Code Repository**
+
+* SST-Macro is the repository for the Sandia developed SST Simulation Macro.  The repository is hosted on [GitHub](https://github.com).
```
Comment (jjwilke) [inline CONTRIBUTING.md:5]: "Sandia-developed macroscale components for the Structural Simulation Toolkit (SST)."

Comment (jjwilke) [inline CONTRIBUTING.md:5]: For information on the discrete event simulation core, see [SST](...sst-core github...)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+#### **Forked Development**
+
+To contribute code to the sst-macro repo, you must create a fork of the repo and setup for forked development.
+
+##### Creating a Fork
+A fork is simply a personal copy of the repo on github for example the official sst-macro repo is located here:
+
+`https://github.com/sstsimulator/sst-macro`
+
+my fork of that repo is here:
```
Comment (jjwilke) [inline CONTRIBUTING.md:36]: I would remove "my fork of that repo is here: https....macro"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+`https://github.com/fryeguy52/sst-macro`
+
+
+The way to create your own fork is to click the fork button in the upper right while you are at the official repo. Once you have done that you can simply clone
```
Comment (jjwilke) [inline CONTRIBUTING.md:41]: Once a user `sst-user` has created a fork, you can clone your fork:

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+```
+git clone https://github.com/fryeguy52/sst-macro
```
Comment (jjwilke) [inline CONTRIBUTING.md:44]: change `fryeguy52` to `sst-user`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+git checkout devel
+git remote add sst-official https://github.com/sstsimulator/sst-macro
+git pull --all
+git branch devel --set-upstream-to sst-official/devel
+```
+You can verify that things are setup correctly
+
+```
+git remote -vv
+origin       https://github.com/fryeguy52/sst-macro (fetch)
```
Comment (jjwilke) [inline CONTRIBUTING.md:61]: Change fryeguy52 to sst-user

**Comment by nmhamster** [comment]
Can someone review this PR - it keeps sitting in testing and is passing just fine (@jjwilke)?

**Comment by jpvandy** [comment]
Hey Stranger,

Are you back?

  John

**Pull request merged**